common/domain/handler.go

```diff
@@ -127,7 +127,8 @@ func (d *handlerImpl) RegisterDomain(
 
 	if !d.clusterMetadata.IsGlobalDomainEnabled() {
 		if registerRequest.IsGlobalDomain {
-			return &types.BadRequestError{Message: "Cannot register global domain when not enabled"}
+			return &types.BadRequestError{Message: "Cannot register global domain when not enabled. Please update config to enable global domain(recommended), " +
```

Comment:
Is this too detail info? the --global_domain belongs to domain CLI?

Reply:
yeah good point. I will remove `--global_domain` from the error message.